lib/samples/snap_geometry_edits/snap_geometry_edits.dart

```diff
@@ -574,9 +586,38 @@ class _SnapGeometryEditsState extends State<SnapGeometryEdits>
     return Column(
       crossAxisAlignment: CrossAxisAlignment.start,
       children: [
-        Text(
-          label,
-          style: Theme.of(context).textTheme.titleMedium,
+        Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            Text(
+              label,
+              style: Theme.of(context).textTheme.titleMedium,
+            ),
+            Row(
+              children: [
+                const Text('Enable all'),
+                // A checkbox to enable all source settings in the category.
+                Checkbox(
+                  value: allSourceSettings.firstWhereOrNull(
+                        (snapSourceSettings) => !snapSourceSettings.isEnabled,
+                      ) ==
+                      null,
```

Comment:
I'm probably a little late, but this could be
```dart
value: allSourceSettings.every((snapSourceSettings) => snapSourceSettings.isEnabled),
```

Reply:
Which saves us the trouble of having `collection` as a dependency, I think

Reply:
Oh nice. I'll pick this back up in a couple of weeks so I can double check and update the dependency list if we can lose it.

Reply:
That worked - flipping the logic! Thanks @PaulAllanSturm , I'll also create a PR to our dependency list.